dash.js

```diff
@@ -840,11 +844,20 @@ var MyDash = GObject.registerClass({
         const nIcons = children.length + addedItems.length - removedActors.length;
         if (nFavorites > 0 && nFavorites < nIcons) {
             if (!this._separator) {
-                this._separator = new St.Widget({
-                    style_class: 'dash-separator',
-                    y_align: Clutter.ActorAlign.CENTER,
-                    height: this.iconSize,
-                });
+                if (!this._isHorizontal) {
+                    this._separator = new St.Widget({
+                        style_class: 'dash-separator',
+                        x_align: Clutter.ActorAlign.CENTER,
+                        width: this.iconSize,
+                    });
+                } else {
+                    this._separator = new St.Widget({
+                        style_class: 'dash-separator',
+                        y_align: Clutter.ActorAlign.CENTER,
```

Comment:
Maybe here you can just use an alternative for the `x_alingn`/`y_align` values depending the dock dispotion

Reply:
Potentially yes, this is an artifact of the vertical implementation being _very_ different in the first pass.

Reply:
https://github.com/micheleg/dash-to-dock/pull/1402/commits/a2d40e2a06117617bcbc5f85608c895c7734604e

Ah now I remember - they use different CSS styles in the latest commit.